spec/ad_parser.spec.js

```diff
@@ -14,9 +20,15 @@ describe('AdParser', () => {
         '<Ad><Wrapper><VASTAdTagURI>foo</VASTAdTagURI></Wrapper></Ad>'
       );
       invalidAdNode = getNodesFromXml('<Ad><Foo></Foo></Ad>');
+      adElement = getNodesFromXml(linearAd);
+    });
+
+    beforeEach(() => {
+      ad = parseAd(adElement, null).ad;
     });
 
     it('correctly returns inline and passes ad attributes down', () => {
+      let inLineAd = parseAd(inlineAdNode, emit);
       expect(parseAd(inlineAdNode, emit)).toEqual({
         ad: expect.objectContaining({ id: 'id-123', sequence: 'seq-123' }),
```

Comment:
you can add `adType` in this `objectContaining`, as it is an attribute to check that gets passed down